octoprint_mrbeam/util/material_csv_parser.py

```diff
@@ -4,42 +4,33 @@
 MRBEAM = "Mr Beam II"
 MRB_DREAMCUT = "MrB II Dreamcut"
 MRB_DREAMCUT_S = "MrB II Dreamcut S"
+MRB_DREAMCUT_x = "MrB II Dreamcut x"
```

Comment:
Why is the "x" not capitalized as in the "S"?

Reply:
because the x should be small from the naming convention